core/services/ardupilot_manager/setup.py

```diff
@@ -13,5 +13,11 @@
     long_description_content_type="text/markdown",
     packages=setuptools.find_packages(),
     python_requires=">=3.6",
-    install_requires=["appdirs==1.4.4", "packaging==20.4", "smbus2==0.3.0"],
+    install_requires=[
+        "appdirs==1.4.4",
+        "packaging==20.4",
+        "smbus2==0.3.0",
+        "fastapi==0.63.0",
+        "uvicorn==0.13.4",
```

Comment:
will this provide a swagger document webpage ?

Reply:
Yes, at port 8000, as specified on main.py